README.md

```diff
@@ -1,10 +1,14 @@
 # aisreporter
-Signal K Node server plugin to report the vessel's AIS data to MarineTraffic and other similar aggregators.
+Signal K Node server plugin to report the vessel's AIS data to MarineTraffic, AISHub and other similar aggregators.
 
-The static info is retrieved from the Signal K full data model. You can use the server's defaults.json mechanism to set the values.
+This plugin generates raw NMEA messages using vessel's position, speed, heading, etc. It does not require an AIS receiver or transceiver. If you do have an AIS, it also doesn't send positions received from it. If you would like a plugin that relays AIS data, check [`ais-forwarder`](https://github.com/hkapanen/ais-forwarder) instead.
+
+The static info used in `aisreporter` is retrieved from the Signal K full data model. You can use the server's defaults.json mechanism to set the values.
```

Comment:
Most if not all fields are editable in the Admin UI Server / Settings.

Reply:
Hey @tkurki , not sure what exactly you would like to highlight here. Why don't you go ahead and make the modifications of what I put in there? Will be easier.

Reply:
👍